presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -670,6 +671,12 @@ public PlanOptimizers(
         builder.add(new HashBasedPartialDistinctLimit(metadata.getFunctionAndTypeManager()));
 
         builder.add(new MetadataDeleteOptimizer(metadata));
+        builder.add(new IterativeOptimizer(
+                ruleStats,
+                statsCalculator,
+                costCalculator,
+                ImmutableSet.of(
+                        new NativeEngineExecutionRewrite())));
```

Comment:
This isn't needed with your changes in the PlanFragmenter, correct? Having the conversion in the PlanFragmenter will be more compatible with AQE.